src/main/webapp/data/userMapData.json

```diff
@@ -278,6 +275,9 @@
       "University of Waterloo",
       "University of Windsor",
       "University of Winnipeg",
+      "Université de Montréal",
+      "Université du QuébecàMontréal",
+      "Université Laval",
```

Comment:
These 3 should probably all be above the "University" group as 'e' (I know it has an accent but still) is before 'y'